### PR TITLE
Iterating stats are removed

### PR DESCRIPTION
**Pull request by muratayan**

### CODE REVIEW

File: hazelcast-ra/hazelcast-jca-rar/pom.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
     <parent>
         <groupId>com.hazelcast</groupId>
         <artifactId>hazelcast-ra</artifactId>
-        <version>3.5.5</version>
+        <version>3.5.5.1</version>
```
Comment (bilalyasar) [inline hazelcast-ra/hazelcast-jca-rar/pom.xml]: should be `3.5.5.2`, i think.
and all pom.xmls' version should be updated

**Comment by hasancelik** [comment]
run-lab-run

**Comment by hasancelik** [comment]
verify

**Comment by hasancelik** [comment]
verify

File: hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java [hunk 2]
```diff
@@ -29,6 +30,7 @@
 public class LocalMapStatsTest extends HazelcastTestSupport {
 
     @Test
+    @Ignore
```
Comment (ahmetmircik) [inline hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java]: can you also add a comment which states the reason of using `@Ignore` for a future reviewer?

**Comment by ahmetmircik** [comment]
:+1:

**Comment by muratayan** [comment]
run-lab-run

**Comment by muratayan** [comment]
verify

**Comment by muratayan** [comment]
verify

**Comment by pveentjer** [comment]
Can you add a description of the purpose of this PR.

File: hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java [hunk 3]
```diff
@@ -28,7 +29,15 @@
 @Category(QuickTest.class)
 public class LocalMapStatsTest extends HazelcastTestSupport {
 
+    /**
```
Comment (pveentjer) [inline hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java]: Isn't it better to change the tests so they reflect the fact that not all information retrieved even though it is present.

**Pull request merged**